sinks/types.go

```diff
@@ -15,8 +15,15 @@
 // This package implements the various storage backends supported by heapster.
 package sinks
 
+import "github.com/GoogleCloudPlatform/heapster/sinks/api/v1"
+
 // TODO: Once in-memory storage is implemented, make the manager extract data from the in-memory store periodically.
-type ExternalSinkManager interface {
+type Sink interface {
 	Store(input interface{}) error
 	DebugInfo() string
 }
+
+type ExternalSinkManager interface {
```

Comment:
ExternalSinkManager was already there, so I didn't touch it. Not sure what the difference between Sink and ExternalSink is, so I didn't change that.